general-concepts/licenses/text.xml

```diff
@@ -39,6 +37,46 @@ be used) then use the following syntax:
 LICENSE="|| ( foo bar )"
 </codesample>
 
+<section>
+<title>License-implied restrictions</title>
+<body>
+
+<p>
+Non-free licenses may require additional restrictions to be stated
+in the ebuild. In order to establish the correct restrictions, it is
+necessary to read the license and establish the applicable clauses
+based on files contained in upstream packages (upstreams may use
+the same license for multiple products, with some restrictions not being
+applicable to the ebuild in question).
```

Comment:
```suggestion
applicable to the ebuild in question.
```

Reply:
I don't really understand what this specifically refers to. If you have a handy example, feel free to append it.